Marlin/Conditionals_LCD.h

```diff
@@ -92,7 +101,7 @@
     #endif
   #endif
 
-  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
+  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER) || ENABLED(ANET_FULL_GRAPHICS_LCD)
```

Comment:
@thinkyhead I don't wan't to remove anything related to the A8 Display. The only thing I would remove are all conditions where the ANET_FULL_GRAPHICS_LCD is added to REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER. Also I would rename the ANET_FULL_GRAPHICS_LCD to REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER on all remaining places where it's used. Maybe it would be good to keep ANET_FULL_GRAPHICS_LCD in the Configuration.h because so much users already have it there and this could be covered by an alias to REPRAP..... . So there would be no need to touch all the files for the boards where this lcd could work, they just can simply use the given REPRAP......